x-pack/plugins/canvas/public/lib/element.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import defaultHeader from './default_header.png';
+
+export class Element {
+  /** The name of the Element. This must match the name of the function that is used to create the `type: render` object  */
+  public name: string;
+  /** A more friendly name for the Element */
+  public displayName: string;
+  /** An image to use in the Element type selector */
+  public image: any;
```

Comment:
Couldn't this be `string` too? It seems like we `CanvasElementConfig.image` is already a string, and we would just need to fix the png import typing to make sure it returns a `string` (if webpack is actually configured like that).

Reply:
Correct!  I addressed this in an update.